midi/midi/encode.py

```diff
@@ -1,28 +1,42 @@
 import os
-# import mido
 import numpy as np
 
 from mido import MidiTrack, MidiFile
 from mido.messages import Message
 from mido.midifiles.meta import MetaMessage
 
-TICKS_IN_BEAT = 0.5
-TEMPO = 937500
+from decode import export_tempo_array, get_array_of_notes, Mode
+
 VELOCITY = 64
+ACCURACY = 64
+
+
+def get_object(filepath):
+    return np.load(filepath, allow_pickle=True)
```

Comment:
Is this needed? I don't see what this function actually does, name is very generic

Reply:
Thank you for your comment.

Reply:
@shilangyu this is obviously an unnecessary function, overlooked before merging - deleted in #19